src/app/cluster/components/AddClusterModal/index.tsx

```diff
@@ -27,7 +27,7 @@ class AddClusterModal extends React.Component<any, any> {
     const { checkConnection, connectionState, name, url, token, mode } = this.props;
 
     return (
-      <Modal isSmall isOpen={this.props.isOpen} onClose={this.handleClose} title="Cluster">
+      <Modal isSmall isOpen={this.props.isOpen} onClose={this.handleClose} title="Add Cluster">
```

Comment:
I think John removes the add because the same modal is used for editing @jwmatthews ?

Reply:
Ah that makes sense. Do we need to remove "Add" from "Add Storage" then? I thought it was odd that they were inconsistent.